CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.0
```

Comment:
I'd make it `2.0.0`, as the switch from `hive` to `sembast` definitely makes the new template version incompatible with the previous one